versions.mk

```diff
@@ -36,6 +39,13 @@ BUILDIMAGE_TAG ?= devel-go$(GOLANG_VERSION)
 BUILDIMAGE ?=  $(DRIVER_NAME):$(BUILDIMAGE_TAG)
 
 GIT_COMMIT ?= $(shell git describe --match="" --dirty --long --always --abbrev=40 2> /dev/null || echo "")
+GIT_COMMIT_SHORT ?= $(shell git rev-parse --short=8 HEAD)
```

Comment:
Should we change this to 7 so that we can copy directly from the github UI?

Reply:
I would love to, across the board. For the reason you mention (and a length of seven is generally regarded to be collision-safe).

But I think we shouldn't discuss this here.

Because we use 8 chars consistently everywhere right now I believe. This variable in particular is used for building  `VERSION_GHCR_CHART` which is used to refer to the chart artifact on GHCR which we [push using 8 chars](https://github.com/NVIDIA/k8s-dra-driver-gpu/blob/13a73595242616e77a2dca6c0122c78ccef1f978/.github/workflows/chart.yaml#L36) in the version specifier.

Reply:
Sure. Was more of a fly-by comment. But I have been slightly annoyed ever time I can't just grab the SHA from the top of the commit list on the main page of the repo and paste it into a script with the chart in it.